src/core/types.jl

```diff
@@ -156,7 +156,7 @@ abstract type AbstractDCPModel <: AbstractActivePowerModel end
 Linearized 'DC' power flow Model with polar voltage variables.
 
 This model is a basic linear active-power-only approximation, which uses branch susceptance values
-`br_b = -br_x / (br_x^2 + br_x^2)` for determining the network phase angles.  Furthermore, transformer
+`br_b = -br_x / (br_x^2 + br_r^2)` for determining the network phase angles.  Furthermore, transformer
```

Comment:
This is a correct fix but the convention is that r comes before x, so `(br_r^2 + br_x^2)` would be more standard.

the reason is the complex number `Z = r + x im`, then `|Z|^2 = r^2 + x^2`